components/CanvasLayout/samples/Dependencies.props

```diff
@@ -11,21 +11,21 @@
 <Project>
     <!-- WinUI 2 / UWP -->
     <ItemGroup Condition="'$(IsUwp)' == 'true'">
```

Comment:
Ah, oops, this is what happened the rebase changed the wrong file for some weird reason... will see if I can fix...